### DISCUSSION TITLE
Turn off Safe Search of YouTube

### DISCUSSION THREAD

**Discussion by NaseerAkbari** [AdguardTeam/AdGuardHome#5657] [General]
Hey guys, 
I have been using this, but my wife is always coming to me and says, most youtube videos are blocked. I dont really want any trouble. 
please someone help me to turn YouTube SafeSearch off. 😁

**Comment by ainar-g** [comment]
In the most recent Edge releases you can enable or disable safe search per-service.  On the current releases, you can add your wife's device as one of persistent clients and turn off safe search completely for her.
